### PR TITLE
Calculate and show frame number

### PR DESCRIPTION
**Pull request by emilk**
### What
Show frame number in selection panel when selecting a video.

image

We may want to make it more prominent though? 🤔

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/8112?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/8112?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/8112)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: crates/store/re_video/src/demux/mp4.rs [hunk 1]
```diff
@@ -84,6 +86,29 @@ impl VideoData {
...
+        {
+            re_tracing::profile_scope!("Sanity-check samples");
+            let mut samples_are_in_decode_order = true;
+            for window in samples.windows(2) {
+                samples_are_in_decode_order &=
+                    window[0].decode_timestamp <= window[1].decode_timestamp;
+            }
+            if !samples_are_in_decode_order {
+                re_log::debug!(
+                    "Video samples are NOT in decode order. This is weird an inefficient."
```
Comment (Wumpf) [inline crates/store/re_video/src/demux/mp4.rs:98]:
```suggestion
                    "Video samples are NOT in decode order. This implies either invalid video data or a bug in parsing the mp4."
```

**Review by Wumpf — APPROVED** [review]
> We may want to make it more prominent though? 🤔

Yeah now that we're confident enough with this (I think we are!) this might be together with the expandy-area in "top-level" 🤔

File: crates/store/re_video/src/demux/mp4.rs (hunk 1 above)
Comment (emilk) [reply crates/store/re_video/src/demux/mp4.rs:98]: Then let's make it a `warn!` too

**Comment by emilk** [comment]
I'll go with this:
image

**Pull request merged**